tests/components/unifi/test_services.py

```diff
@@ -9,26 +13,21 @@
 from homeassistant.core import HomeAssistant
 from homeassistant.helpers import device_registry as dr
 
-from .test_hub import setup_unifi_integration
-
 from tests.test_util.aiohttp import AiohttpClientMocker
 
 
+@pytest.mark.parametrize(
+    "client_payload", [[{"is_wired": False, "mac": "00:00:00:00:00:01"}]]
+)
 async def test_reconnect_client(
     hass: HomeAssistant,
     device_registry: dr.DeviceRegistry,
     aioclient_mock: AiohttpClientMocker,
+    config_entry_setup: ConfigEntry,
```

Comment:
The imports are missing for the new type hints

Reply:
Fixed